modules/ROOT/pages/index.adoc

```diff
@@ -20,6 +20,49 @@ With Mule you can:
 * Use error constructs and try scopes for rapid debugging.
 * Isolate the classloader to protect Mule apps from changes to the runtime or connectors.
 
+[[install_deploy_manage]]
+== Installation, Deployment, Management
+
+For a Mule app to run, it must be deployed into an environment where the Mule
+runtime engine is installed.
+
+Mule is packaged within the xref:7.3@studio::index.adoc[Studio] IDE and with
+xref:design-center::index.adoc[Design Center] on Anypoint Platform so
+that you can run a Mule app as you design it.
+
+For production and pre-production deployments of Mule apps, you can use
+xref:runtime-manager::index.adoc[Runtime Manager] to deploy Mule apps
+to runtimes within CloudHub and other supported platform as a service (PaaS)
+solutions.
+
+* xref:runtime-manager::cloudhub.adoc[CloudHub] is a fully managed, Cloud-based
+integration platform as a service (iPaaS) for Anypoint Platform that enables you
+to run your Mule apps without requiring you to provide Mule runtimes or the infrastructure
+on which your apps run. You use Runtime Manager to deploy Mule apps to CloudHub,
+select the Mule runtime version, set the number of vCores needed to run the app,
+and so on.
+* Hybrid deployment models manage Mule apps and runtimes from the Cloud while
+running them in a datacenter that is managed by your company:
+** For remote Mule runtimes (also called standalone or "naked" Mules), you
+start Mule runtimes from your datacenter, but you can deploy and manage
+Mule apps from the Cloud, though Runtime Manager. In this deployment model, you
+provide the infrastructure and Mule runtime (see
+xref:mule-standalone.adoc[Mule Installation]).
+** For a hybrid PaaS deployment, you set up and run the PaaS on your company's
+datacenter and use Runtime Manager to manage Mule apps within the PaaS. In this
+case, you provision the infrastructure in which the apps run. To guarantee
+high availability of Mule apps, you use Runtime Manager to handle Mule runtimes.
```

Comment:
I think it sounds better to say "the high availability of Mule apps

Reply:
done